### PR TITLE
Working down through the list of errors, another one eliminated

### PR DESCRIPTION
**Pull request by JJ**
Also some minor warnings with the previous one. I would say this one is ready for CPAN :-)
Thanks for accepting the other PRs.

### CODE REVIEW

File: Info.pm [hunk 1]
```diff
@@ -652,7 +652,8 @@ sub _get_v1tag {
 				# Remove garbage and retry
 				# (string is truncated in the
 				# middle of a multibyte char?)
-				$value =~ s/(.)$//;
+				#$value =~ s/(.)$//; # error https://rt.cpan.org/Ticket/Display.html?id=101620
```
Comment (dsully) [inline Info.pm]: Just have the http link here, not the commented out code.

Comment (JJ) [reply Info.pm]: It's right below, line 656 
screenshot from 2017-06-11 09 28 40

Comment (dsully) [reply Info.pm]: Yes, I'm saying remove the commented out '$value =~ s/(.)$//;' as it provides no value. Leave the comment pointing to the RT issue.

Comment (JJ) [reply Info.pm]: Sorry, I didn't understand you at first. Done now.

**Comment by JJ** [comment]
Could you please release this new version to CPAN?

**Pull request merged**